src/common/base/src/mem_allocator/mod.rs

```diff
@@ -12,12 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-mod allocators;
+mod je_allocator;
```

Comment:
Maybe we can add a new crate called `common-allocator` in the following PRs?

Cc @PsiACE

Reply:
Tracked at https://github.com/datafuselabs/databend/issues/7223